certified-connectors/CyberProof/apiDefinition.swagger.json

```diff
@@ -30,43 +30,153 @@
     }
   },
   "paths": {
-	"/api/v1/actions/views/packs": {
```

Comment:
@ejaz-shaikh-cp The operation has been deleted. This change breaks existing flows. can you please confirm that no existing flows and users will be affected by this change? I would suggest you to [deprecate ](https://learn.microsoft.com/en-us/connectors/custom-connectors/deprecate-operation)this operation.

Reply:
@Amjed-Ayoub  This change was never published on logic app connector so no user is using this operation hence we have deleted this.

Reply:
Thank you @ejaz-shaikh-cp for the confirmation.